### DISCUSSION TITLE
this-week-in-neovim/63

### DISCUSSION THREAD

**Discussion by giscus** [codicocodes/dotfyle#119] [This Week in Neovim]
# this-week-in-neovim/63

This Week in Neovim 63 with news and updates from the Neovim plugin ecosystem and Neovim core.

https://dotfyle.com/this-week-in-neovim/63

**Comment by ca-mantis-shrimp** [comment]
Thank you for making these every week.

Your contributions are invaluable and I hope you have a long, happy vacation.

**Reply by codicocodes** [reply]
Thank you! I really appreciate it!

**Comment by dkarter** [comment]
This was great! Thank you for keeping this alive and better than ever!
